modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+
+module ClaimsApi
+  class SpecialIssueUpdater
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform(user, special_issues, auto_established_claim)
+      service = bgs_service(user).contention
+
+      contentions = service.find_contentions_by_ptcpnt_id(user.participant_id)
+
+      contention = contentions[:benefit_claims].select do |claim|
+                     claim[:clm_id] == auto_established_claim&.evss_id.to_s
+                   end [0]
+
+      options = contention_options(contention)
+
+      options[:contentions] =
+        [{
+          clm_id: contention[:clm_id],
+          special_issues: special_issues.map { |special_issue| { spis_tc: special_issue } }
+        }]
```

Comment:
im not sure this is what we want here, it seems it would blow away all the other contentions currently on the Veteran

Reply:
you're right, it would. I don't think that would be an issue for updating special issues on a new claim, but it would for an _existing_ contention that had special issues. given some edge cases I outlined [here](https://vajira.max.gov/browse/API-3272?focusedCommentId=770924&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-770924), it might be more sense to have a separate method to update special issues for existing contentions.